runtime/include/vendor_version.h

```diff
@@ -32,3 +32,10 @@
  * #define VENDOR_SHORT_NAME "ABC"
  * #define VENDOR_SHA "1a2b3c4"
  */
+
```

Comment:
Please document VENDOR_VERSION_STRING in the comments above.

Reply:
Updated.